ansible_bender/core.py

```diff
@@ -205,10 +205,18 @@ def build(self, db_path):
             tmp_pb_path = os.path.join(tmp, "p.yaml")
             with open(self.pb, "r") as fd_r:
                 pb_dict = yaml.safe_load(fd_r)
+                host_present = False
             for idx, doc in enumerate(pb_dict):
-                host = doc["hosts"]
-                logger.debug("play[%s], host = %s", idx, host)
-                doc["hosts"] = self.builder.ansible_host
+                try:
+                    host = doc["hosts"]
+                    host_present = True
+                except:
+                    continue
```

Comment:
looking at my code, I can see that the host is retrieved only for logging purposes, so I'd instead do something like:
```python
try:
  host = doc["hosts"]
except KeyError:
  host = "not set"
```
and continue with the former flow